_events/sr2023/southampton-kickstart.md

```diff
@@ -43,3 +43,9 @@ Southampton's information on [getting to our campuses][soton-campus-directions]
 
 [teams-contact]: mailto:teams@studentrobotics.org
 [soton-campus-directions]: https://www.southampton.ac.uk/student-life/campuses/highfield
+
+## Livestream
+
+Our livestream will start at <time datetime="2023-10-22T10:00:00+01:00" title="Sun, 22 Oct 2023 10:00:00 +0100">10am UK time</time>: [Watch live](https://www.youtube.com/watch?v=FVnNv5GSpyQ)
```

Comment:
Should we note what's planned to be covered in the stream? That is: that it's just the presentation?